serialization/InformationModel/README.md

```diff
@@ -0,0 +1,307 @@
+(spdx-tools-3.0.1), and a Person (iamwillbar). The first task is to determine the logical values of these
+three elements. 
+
+The logical values of these elements are:
+
+**Package:**
+```json
+{
+  "id": "urn:spdx.dev:spdx-tools-3.0.1",
+  "type": {
+    "package": {
+      "packagePurpose": ["application"],
+      "downloadLocation": "https://spdx.dev/downloads/spdx-tools-3.0.1.tgz",
+      "homePage": "https://spdx.dev/tools.3.0",
+      "originator": ["urn:spdx.dev:project"]
+    }
+  },
+  "externalIdentifier": [
+    {"type": "purl", "identifier": ""},
+    {"type": "cpe_2.2", "identifier": ""}
+  ],
+  "verifiedUsing": [{"hash": {"sha256": "14a657a7118a333cc1fdc6af05071a59cda067fd11130d4ee5d6d47c26e7863f"}}],
+  "creator": ["urn:spdx.dev:iamwillbar"],
+  "created": "2022-05-02T20:28:00.000Z",
+  "specVersion": "3.0",
+  "profile": ["core", "software"],
+  "dataLicense": "CC0-1.0"
+}
+```
+**Person:**
+```json
+{
+  "id": "urn:spdx.dev:iamwillbar",
+  "type": {
+    "person": {
+      "identifiedBy": [
+        {"email": "willbar@microsoft.com"},
+        {"account": {"authority": "github.com", "localId": "iamwillbar"}}
+      ]
+    }
+  },
+  "name": "William Bartholomew",
+  "creator": ["urn:spdx.dev:iamwillbar"],
+  "created": "2022-05-02T20:28:00.000Z",
+  "specVersion": "3.0",
+  "profile": ["core"],
+  "dataLicense": "CC0-1.0"
+}
+```
+The `person` element illustrates a problem with the logical model that can be resolved by adding
+an `identifiedBy` property to the Identity type. For the purpose of illustrating serialization, assume that the
+problem has been resolved in this manner.
+
+**Sbom:**
+```json
+{
+  "id": "urn:spdx.dev:null-sbom",
+  "type": {
+    "sbom": {
+      "element": [
+        "urn:spdx.dev:iamwillbar",
+        "urn:spdx.dev:spdx-tools-3.0.1",
+        "urn:spdx.dev:project",
+        "urn:spdx.dev:doc"
+      ],
+      "import": [
+        {
+          "externalId": "urn:spdx.dev:project",
+          "verifiedUsing": {"hash": {"sha256": "14a657a7118a333cc1fdc6af05071a59cda067fd11130d4ee5d6d47c26e7863f"}},
+          "locationHint": "https://spdx.dev/projects/v1.0.json"
+        },
+        {
+          "externalId": "urn:spdx.dev:doc",
+          "verifiedUsing": {"hash": {"sha256": "14a657a7118a333cc1fdc6af05071a59cda067fd11130d4ee5d6d47c26e7863f"}},
+          "locationHint": "https://spdx.dev/docs/v1.0.json"
+        }
+      ]
+    }
+  },
+  "creator": ["urn:spdx.dev:iamwillbar"],
+  "created": "2022-05-02T20:28:00.000Z",
+  "specVersion": "3.0",
+  "profile": ["core", "software"],
+  "dataLicense": "CC0-1.0"
+}
+
+```
+Note that every JSON structure has both leaf-path (RFC 6901) and hierarchical representions. The two represent
+identical values and can be converted back and forth without any semantic knowledge.
+For example, the path representation of the Sbom element is:
+
+**Sbom (path):**
+```json
+{
+  "id": "urn:spdx.dev:null-sbom",
+  "type/sbom/element/0": "urn:spdx.dev:iamwillbar",
```

Comment:
Could you include a fragment in one of the URN examples (e.g. `urn:spdx.dev:people#iamwillbar` just to show it is possible and acceptable?  Important for converting from SPDX 2.X.

Reply:
Yes.  A payload and the SpdxDocument element that describes it will have a namespace.  If that Document/Payload contains only Person elements, the namespace could be `urn:spdx.dev:people#` and each Person IRI would be of the form `iamwillbar`.  If the payload has elements of other types (individually typed), the payload namespace could be `urn:spdx.dev:` and the Person IRIs would be `person#iamwillbar`, `package#ctime`, etc.